cartography/intel/gcp/__init__.py

```diff
@@ -296,7 +296,8 @@ def _services_enabled_on_project(serviceusage: Resource, project_id: str) -> Set
 
 def concurrent_execution(
     service: str, service_func: Any, config: Config, iam: Resource,
-    common_job_parameters: Dict, gcp_update_tag: int, project_id: str, crm: Resource,
+    common_job_parameters: Dict, gcp_update_tag: int, project_id: str, crm_v1: Resource,
```

Comment:
is it possible to use only 1 version of crm, the v2? Instead of having 2 different resources for v1 & v2.

Reply:
v2 has only folder api and v1 doesn’t contain folder api so we need to use both